Remove modules that require software not yet ported to Solaris, such as
Network Manager.

Even if we ported all their dependencies, it's not clear we want these modules,
since they are focused on setting up a new machine, not a new user - we have
our initial user creation & network configuration handled by our OS install
software instead.

Disabling the timezone module for the moment as it causes gnome-initial-setup to
core in tz_info_get_clean_name.  We can look into enabling it, if needed, once 
all other modules that gnome-initial-setup depends on are updated to the latest.

For the dependency checks we've removed:
 - libnm & libnma are not needed because we don't build the network page
 - gweather, geoclue, & geocode-glib are not needed because we don't build
   the timezone page
 - pango is not needed because we don't build the account or timezone pages
 - librest is not needed because we don't build the account page
 - json-glib is not needed because we don't build the account page
 - kerberos is not needed because we don't build the account page

Not appropriate for upstream in it's current hardcoded form - would have
to make meson options for each of these to upstream.

--- gnome-initial-setup-41.4/gnome-initial-setup/gnome-initial-setup.c
+++ gnome-initial-setup-41.4/gnome-initial-setup/gnome-initial-setup.c
@@ -68,13 +68,9 @@ static PageData page_table[] = {
   PAGE (welcome, FALSE),
   PAGE (language, FALSE),
   PAGE (keyboard, FALSE),
-  PAGE (network,  FALSE),
   PAGE (privacy,  FALSE),
-  PAGE (timezone, TRUE),
   PAGE (software, TRUE),
   PAGE (goa,      FALSE),
-  PAGE (account,  TRUE),
-  PAGE (password, TRUE),
 #ifdef HAVE_PARENTAL_CONTROLS
   PAGE (parental_controls, TRUE),
   PAGE (parent_password, TRUE),
--- gnome-initial-setup-41.4/gnome-initial-setup/meson.build
+++ gnome-initial-setup-41.4/gnome-initial-setup/meson.build
@@ -28,29 +28,19 @@ 'gis-keyring.h'
 ]
 
 dependencies = [
-    dependency ('libnm', version: '>= 1.2'),
-    dependency ('libnma', version: '>= 1.0'),
     dependency ('polkit-gobject-1', version: '>= 0.103'),
     dependency ('accountsservice'),
     dependency ('gnome-desktop-3.0', version: '>= 3.7.5'),
     dependency ('gsettings-desktop-schemas', version: '>= 3.37.1'),
     dependency ('fontconfig'),
-    dependency ('gweather-3.0'),
     dependency ('goa-1.0'),
     dependency ('goa-backend-1.0'),
     dependency ('gtk+-3.0', version: '>= 3.11.3'),
     dependency ('glib-2.0', version: '>= 2.63.1'),
     dependency ('gio-unix-2.0', version: '>= 2.53.0'),
     dependency ('gdm', version: '>= 3.8.3'),
-    dependency ('geocode-glib-1.0'),
-    dependency ('libgeoclue-2.0', version: '>= 2.3.1'),
     cc.find_library('m', required: false),
-    dependency ('pango', version: '>= 1.32.5'),
-    dependency ('rest-0.7'),
-    dependency ('json-glib-1.0'),
-    dependency ('krb5'),
     dependency ('libsecret-1', version: '>= 0.18.8'),
-    dependency ('pwquality'),
     dependency ('webkit2gtk-4.0', version: '>= 2.26.0'),
     cheese_dep,
     cheese_gtk_dep,
--- gnome-initial-setup-41.4/gnome-initial-setup/pages/meson.build
+++ gnome-initial-setup-41.4/gnome-initial-setup/pages/meson.build
@@ -1,12 +1,8 @@
 pages = [
-   'account',
    'language',
    'keyboard',
-   'network',
-   'timezone',
    'privacy',
    'goa',
-   'password',
    'software',
    'summary',
    'welcome',
